tests/openebs-pool-container-failure_test.go

```diff
@@ -80,10 +82,45 @@ var _ = BeforeSuite(func() {
 var _ = Describe("BDD of openebs pool container failure experiment", func() {
 
 	// BDD TEST CASE 1
+	rv1 := 0
+	rc1 := 0
 	Context("Check for the components of openebs", func() {
 
 		It("Should check for creation of runner pod", func() {
 
+			//Getting the Sum of Resource Version before Chaos
+			csp_rv, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get csp pods")
+			for _, v := range csp_rv.Items {
+				i1, _ := strconv.Atoi(v.ResourceVersion)
+				rv1 = rv1 + i1
+			}
+
+			fmt.Printf("Resource Version before chaos has been recorded\n")
+
+			//Getting the ContainerIDs of CSP pod Containers
+			k := 0
+			csp, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get csp pods")
+			for _, v := range csp.Items {
+				for i := 0; i < 3; i++ {
+					container_id1[k] = (v.Status.ContainerStatuses[i].ContainerID)
+					k++
+				}
+			}
+
+			fmt.Printf("ContainerIDs before chaos has been recorded\n")
+
+			//Getting the Sum of Container Restart Count
+			csp_rc, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
+			Expect(err).To(BeNil(), "fail to get csp pods")
+			for _, v := range csp_rc.Items {
+				for i := 0; i < 3; i++ {
+					rc1 = rc1 + int(v.Status.ContainerStatuses[i].RestartCount)
```

Comment:
Why are you writing an extra loop for this task? You can perform this action while storing container ID. It may save lines & time both

Reply:
The first loop is for iterating over pods and second for the container in the pod.